src/components/caption/caption.tsx

```diff
@@ -113,42 +98,34 @@ export class Caption extends Component<ExtendedCaptionProps, CaptionState> {
     );
   };
 
-  private _handleFocus = () => {
-    this.setState({
-      focused: true
-    });
-  };
-
-  private _handleBlur = () => {
-    this.setState({
-      focused: false
-    });
-  };
-
   render() {
     const {caption, highlighted, showTime, longerThanHour} = this.props;
     const {startTime, id} = caption;
     const isHighlighted = Object.keys(highlighted)[0] === id;
+    const time = showTime ? secontsToTime(startTime, longerThanHour) : '';
 
     const captionA11yProps: Record<string, any> = {
       ariaCurrent: isHighlighted,
-      onFocus: this._handleFocus,
-      onBlur: this._handleBlur,
       tabIndex: 0,
-      ariaHidden: !(isHighlighted || this.state.focused)
+      ariaLabel: `${time}${showTime ? ' ' : ''}${caption.text}`,
+      role: 'button'
     };
 
     return (
-      <A11yWrapper onClick={this._handleKeyDown} role={isHighlighted ? 'text' : 'listitem'}>
+      <A11yWrapper onClick={this._handleClick}>
         <div
           className={styles.caption}
           ref={node => {
             this._hotspotRef = node;
           }}
           {...captionA11yProps}>
-          {showTime && <div className={styles.captionTime}>{secontsToTime(startTime, longerThanHour)}</div>}
+          {showTime && (
+            <div className={styles.captionTime} aria-hidden="true">
```

Comment:
why aria-hidden ? this element shouldn't be exposed ?

Reply:
yes, it shouldn't be exposed. The parent node (button that user can click) has aria label with the data and screen-reader reads the captionTime and captionContent.
Current approach I took from youtube transcript feature.